omnibridge/main.py

```diff
@@ -1,96 +1,40 @@
-from omnibridge.saved_data.json_data_manager import JsonDataManager
-from omnibridge.wrappers.api_key import ApiKey
 from omnibridge.cli.banner import banner
-
-from omnibridge.wrappers.wrapper_instances.dalle_wrapper import DALLEWrapper
-from omnibridge.wrappers.wrapper_instances.gpt_wrapper import GPTWrapper
+from omnibridge.cli.argument_parser.argument_parser import add_create_key_sub_parser, \
+    add_create_model_sub_parser, add_create_flow_sub_parser, add_run_flow_sub_parser, \
+    add_list_wrappers_arguments
+from omnibridge.cli.argument_parser.command_handlers import handle_create_command, \
+    handle_list_wrappers_command, handle_run_flow_command
+from omnibridge.saved_data.json_data_manager import FILE_PATH
 from omnibridge.wrappers.wrapper_instances.type_name_to_wrapper import ModelLoader
 
+SAVED_DATA_FILE_PATH = os.getenv('SAVED_DATA_FILE_PATH', FILE_PATH)
 
 def run() -> int:
     parser = argparse.ArgumentParser(description='AI integration tool.')
     parser.add_argument('-p', '--prompt', help="prompt for model", type=str, default=[])
-    parser.add_argument('-l', "--load-config", help="absolute path to models configuration file")
     parser.add_argument('-n', "--model-name", help="which model to use.")
 
     subparsers = parser.add_subparsers(help='sub-command help', dest='command')
-    api_key_parser = subparsers.add_parser("add-key", help="Add an api key.")
-    api_key_parser.add_argument('-n', '--name', help="name of the api key.", type=str, required=True)
-    api_key_parser.add_argument('-v', '--value', help="value of the api key.", type=str, required=True)
-
-    add_chatgpt_model_parser = subparsers.add_parser("add-chatgpt", help="add chatgpt model connection details.")
-    add_chatgpt_model_parser.add_argument('-n', '--name', type=str, required=True,
-                                          help="name of the model, e.g. my_gpt4.")
-    add_chatgpt_model_parser.add_argument('-k', '--key', type=str, required=True, help="api key name.")
-    add_chatgpt_model_parser.add_argument('-m', '--model', type=str, default="gpt-3.5-turbo",
-                                          help="model, e.g. 3.5 or 4.")
-
-    add_chatgpt_model_parser = subparsers.add_parser("add-dalle", help="add dalle model.")
-    add_chatgpt_model_parser.add_argument('-n', '--name', type=str, required=True, help="name of the model.")
-    add_chatgpt_model_parser.add_argument('-k', '--key', type=str, required=True, help="api key name.")
-    add_chatgpt_model_parser.add_argument('--num-images', type=str, default="4",
-                                          help="number of images per prompt, default 4.")
-    add_chatgpt_model_parser.add_argument('-r', '--res', type=str, default="256x256", help="resolution of images.")
-
+    create_parser = subparsers.add_parser("create", help="Create models/keys/flows.")
+    create_subparsers = create_parser.add_subparsers(help='create sub-command help', dest='create_command')
```

Comment:
is 'create_command' name suites here? it may be confused with the "create" keyword itself used for creating, maybe it will be better using "created_object" or some other variance. btw chatgpt helps me a lot for name picking.

Reply:
changed to `object_to_create`